storage/pebble/lookup.go

```diff
@@ -14,13 +14,38 @@ type lookupKey struct {
 	encoded []byte
 }
 
+func createHeightKey(identifier byte) []byte {
+	key := make([]byte, 0, MinLookupKeyLen)
+	key = append(key, identifier)
+	key = append(key, '/')
+	key = binary.BigEndian.AppendUint64(key, placeHolderHeight)
+	return key
+}
+
+// LatestHeightKey is a special case of a lookupKey
+// with keyLatestBlockHeight as key, no owner and a placeholder height of 0.
+// This is to ensure SeekPrefixGE in pebble does not break
+func LatestHeightKey() []byte {
```

Comment:
Any reason these are exported?

Reply:
yes! any other modules that use pebble may need to access this, but I can make it back to private for now